src/test/components/__snapshots__/MenuButtons.test.js.snap

```diff
@@ -1627,12 +1627,24 @@ exports[`app/MenuButtons <Publish> matches the snapshot for the menu buttons and
     <div
       class="menuButtonsPublishButtons"
     >
-      <button
-        class="photon-button menuButtonsPublishButton menuButtonsPublishButtonsDownload menuButtonsPublishButtonDisabled"
-        type="button"
+      <a
+        class="photon-button menuButtonsPublishButton menuButtonsPublishButtonsDownload"
+        download="Firefox 1970-01-01 00.00 profile.json.gz"
+        href="mockCreateObjectUrl"
       >
-        Compressing…
-      </button>
+        <span
+          class="menuButtonsPublishButtonsSvg menuButtonsPublishButtonsSvgDownload"
+        />
+        Download
+         
+        <span
+          class="menuButtonsDownloadSize"
+        >
+          (
+          9 B
+          )
+        </span>
+      </a>
```

Comment:
I haven't looked closely, but I _think_ we get this snapshot difference because we're waiting longer. Previously we were doing a synchronous test with the fake timers, but the gzipping isn't synchronous so we would never finish it, but now we're finishing it. Hopefully this won't be intermittent, but if this happens we can change the test to use `await findByText('Download')` for example.

Reply:
Yeah, I guess it makes sense that it's waiting longer now. I tried a few times on my machine and don't see an issue. I hope it will not be in the future as well.